test/client/spec/victory-util/prop-types.spec.js

```diff
@@ -254,4 +255,24 @@ describe("prop-types", () => {
       expect(result).not.to.be.an.instanceOf(Error);
     });
   });
+
+  describe("matchDataLength", () => {
+    const validate = function (prop, dataProp) {
+      const props = {testProp: prop, data: dataProp};
+      return matchDataLength(props, "testProp", "TestComponent");
+    };
+
+    it("does not return an error when prop is undefined", () => {
+      expect(validate()).to.not.be.an.instanceOf(Error);
+    });
+
+    it("does not return an error when prop has same length as data", () => {
+      expect(validate([{}, {}], [1, 2])).to.not.be.an.instanceOf(Error);
+    });
+
+    it("returns an error when prop doesn't have same length as data", () => {
+      expect(validate([{}], [1, 2])).to.be.an.instanceOf(Error)
+        .and.have.property("message", "Length of data and testProp arrays must match.");
```

Comment:
This isn't enforced by ESLint, so it may not matter.  But I know
```
expect(thing)
  .to.be.instanceOf(Error).and
  .to.be.otherThing;
```
has been the preferred style in the past.